IronSoftware.Drawing/IronSoftware.Drawing.Common.Tests/IronSoftware.Drawing.Common.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net472;netcoreapp3.1;net50;net60;net70</TargetFrameworks>
+		<TargetFrameworks>net472;net80</TargetFrameworks>
```

Comment:
Can we do .NET Framework 4.8 instead of 4.7.2? I think it would make sense since there are more customers using this one.